projects/js-packages/publicize-components/src/social-store/selectors/pricing-page.ts

```diff
@@ -0,0 +1,19 @@
+import { store as coreStore } from '@wordpress/core-data';
+import { createRegistrySelector } from '@wordpress/data';
+import { getSocialScriptData } from '../../utils';
+import { SHOW_PRICING_PAGE_KEY } from '../constants';
+
+/**
+ * Returns the UTM state.
+ */
+export const getShouldShowPricingPage = createRegistrySelector( select => () => {
```

Comment:
Why use both `get` and `should`?

Why not simply?

```suggestion
export const shouldShowPricingPage = createRegistrySelector( select => () => {
```

Also, the description needs to be updated - `Returns the UTM state.`

Reply:
Fixed in 44f8c61, thanks!